src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -305,6 +306,27 @@ public boolean apply(StepDescriptor i) {
         return r;
     }
 
+    /** Return true if we can easily create a nice String for user display from the object */
+    static boolean isAbleToUseToStringForDisplay(@CheckForNull Object o) {
+        return (o != null && (o instanceof Comparable || o instanceof CharSequence)); // Covers our base types
```

Comment:
`Comparable` seems dodgy to me, there is no contract with `Comparable` that will ensure general utility... I think `Number` or `Boolean` is really what you want for this, e.g. `(o instanceof CharSequence || o instanceof Number || o instanceof Boolean)`